### PR TITLE
Added SVG placeholder icons

### PR DESCRIPTION
**Pull request by MichaelArestad**
* includes changes to Gruntfile.js
* `grunt` now runs the Watch task (currently only watches SVG files)
* `grunt lint` runs linters

### CODE REVIEW

**Comment by MichaelArestad** [comment]
If there's a preference on which folder font/icon/other assets should live, I'll move the folders there and update the PR.

File: Gruntfile.js [hunk 1]
```diff
@@ -33,6 +85,7 @@ grunt.registerTask( "update-authors", function() {
 	});
 });
 
-grunt.registerTask( "default", [ "jshint", "jscs" ] );
+grunt.registerTask( "default", [ "watch" ] );
```
Comment (arschmitz) [inline Gruntfile.js]: I would not have watch run on the default task I never ever use watch. Im all for having a watch task just not as the default.

File: Gruntfile.js [hunk 2]
```diff
@@ -13,6 +16,55 @@ grunt.initConfig({
...
+				ext: ".svg"
+			} ]
+		}
+	},
+
+	// Combines SVGs into single file
+	svgstore: {
+		defaults: {
+			options: {
+				prefix: "icon-", // This will prefix each ID
```
Comment (arschmitz) [inline Gruntfile.js]: We previously decided to follow the jQuery js style guide unless we come up with a strong reason not to http://contribute.jquery.org/style-guide/js/

So comments should be on the line above with a blank line preceding them here and line below

File: Gruntfile.js (hunk 1 above)
Comment (MichaelArestad) [inline Gruntfile.js]: I can remove it making it `grunt watch` instead, but it's pretty handy when working with Sass/LiveReload (when we get that set up). `

Comment (arschmitz) [inline Gruntfile.js]: Yeah i'm all for having a watch task

File: Gruntfile.js (hunk 2 above)
Comment (MichaelArestad) [inline Gruntfile.js]: Rad. Changing.

**Comment by arschmitz** [comment]
Also can I get you to sign the CLA at http://contribute.jquery.org/CLA/ Please :-)

**Comment by MichaelArestad** [comment]
Done. :)

**Comment by arschmitz** [comment]
A personal preference thing here and ill let other weigh in but i don't like to see dist or min files in repos. I would make a build task and have that run on the default task. Cuts down own bloat in the repo and also avoids issues with issues things being updated for fixed but the update to dist files getting missed. ( iv seen this a lot with minor fixes like typos )

**Comment by MichaelArestad** [comment]
Works for me. (same preference) Removing the min/build files.

**Comment by sfrisk** [comment]
Does anyone have any other concerns about this PR?

**Comment by arschmitz** [comment]
Looks good to me :+1:

**Comment by MichaelArestad** [comment]
Works for me. I can add fallback bits in another PR.

**Comment by arschmitz** [comment]
@MichaelArestad oh sorry just went to land this, and realized you are using your s/n name, instead of your real name, in your git config. Think you could fix that? ( I know its just a space but it needs to match to pass the checks )

**Pull request closed without merging**